src/Persistent_cohomology/include/gudhi/Persistent_cohomology.h

```diff
@@ -208,6 +208,31 @@ class Persistent_cohomology {
     }
   }
 
+  /**
+   * @private
+   * Temporary patch to make the computation of more general FilteredComplex possible
+   * Will be removed when a better solution was agreed on.
+   */
+  void compute_persistent_cohomology_without_optimizations(Filtration_value min_interval_length = 0) {
+    if (dim_max_ <= 0)
```

Comment:
I guess we don't generalize to the point of allowing cells with negative dimension? I don't know if that could ever appear or if it would cause trouble somewhere in the code.